docs/switching_from/conda.md

```diff
@@ -0,0 +1,83 @@
+# Transitioning from the `conda` or `mamba` to `pixi`
+Welcome to the guide designed to ease your transition from `conda` or `mamba` to `pixi`.
+This document compares key commands and concepts between these tools, highlighting `pixi`'s unique approach to managing environments and packages.
+With `pixi`, you'll experience a project-based workflow, enhancing your development process, and allowing for easy sharing of your work.
+
+## Why Pixi?
+
+`Pixi` builds upon the foundation of the conda ecosystem, introducing a project-centric approach rather than focusing solely on environments.
+This shift towards projects offers a more organized and efficient way to manage dependencies and run code, tailored to modern development practices.
+
+## Key Differences at a Glance
+
+| Task                        | Conda/Mamba                                       | Pixi                                                                 |
+|-----------------------------|---------------------------------------------------|----------------------------------------------------------------------|
+| Installation                | Requires an installer                             | Download and add to path (See [installation](../index.md))           |
+| Creating an Environment     | `conda create -n myenv -c conda-forge python=3.8` | `pixi init myenv` followed by `pixi add python=3.8`                  |
+| Activating an Environment   | `conda activate myenv`                            | `pixi shell` within the project directory                            |
+| Deactivating an Environment | `conda deactivate`                                | `exit` from the `pixi shell`                                         |
+| Running a Task              | `conda run -n myenv python my_program.py`         | `pixi run python my_program.py` (See [run](../reference/cli.md#run)) |
+| Installing a Package        | `conda install numpy`                             | `pixi add numpy`                                                     |
+| Uninstalling a Package      | `conda remove numpy`                              | `pixi remove numpy`                                                  |
+
+!!! warn "No `base` environment"
+    Conda has a base environment, which is the default environment when you start a new shell.
+    **Pixi does not have a base environment**. And requires you to install the tools you need in the project or globally.
+    Using `pixi global install bat` will install `bat` in a global environment, which is not the same as the `base` environment in conda.
+
+??? tip "Activating pixi environment in the current shell"
+    For some advanced use-cases, you can activate the environment in the current shell.
+    This uses the `pixi shell-hook` which prints the activation script, which can be used to activate the environment in the current shell without `pixi` itself.
+    ```shell
+    ~/myenv > eval "$(pixi shell-hook)"
```

Comment:
I'd emphasize `pixi shell`, possibly with it's own section over using `shell-hook`. 

It would also be worth showing how `--manifest-path` can be used to point to activate a shell from a different directory.